cyences_app_for_splunk/default/commands.conf

```diff
@@ -81,3 +81,7 @@ filename = cyences_upgrade.py
 chunked = true
 python.version = python3
 
+[azureadfieldextraction]
```

Comment:
field_extraction name seems bit generic, can we do something like **modified properties formatter** or something?